performance/objcinterop/src/nativeInterop/cinterop/complexNumbers.h

```diff
@@ -0,0 +1,23 @@
+#import <Foundation/Foundation.h>
+
+@protocol CustomNumber
+@required
+- (id<CustomNumber>)add: (id<CustomNumber>)other;
+- (id<CustomNumber>)sub: (id<CustomNumber>)other;
+@end
+
+@interface Complex : NSObject<CustomNumber>
+
+@property (nonatomic, readonly) double re;
+@property (nonatomic, readonly) double im;
+@property (retain, nonatomic, readwrite) NSString *format;
+
+- (id)initWithRe: (double)re andIm: (double)im;
++ (Complex *)complexWithRe: (double)re im: (double)im;
+- (void)setFormat: (NSString *)newFormat;
```

Comment:
This method is not required, it is generated automatically for `readwrite` property.